src/dataLoader/savedItemsDataLoader.ts

```diff
@@ -35,23 +17,10 @@ export async function batchGetSavedItemsByUrls(
     context
   ).batchGetSavedItemsByGivenUrls(urls);
 
-  return reorderSavedItemsByUrls(urls, savedItems);
-}
-
-/**
- * Reorders savedItems based on the id that are received to
- * keep the response in the same order of savedItemIds as requested
- * by the Apollo Gateway (Client API)
- * @param ids
- * @param savedItems
- */
-export function reorderSavedItemsByIds(ids: string[], savedItems: SavedItem[]) {
-  const idToSavedItemMap = savedItems.reduce((acc, savedItem) => {
-    acc[savedItem.id] = savedItem;
-    return acc;
-  }, {});
-
-  return ids.map((id) => idToSavedItemMap[id]);
+  return reorderResultByKey<SavedItem, 'url'>(
```

Comment:
consolidating to use the generic function

Reply:
having trouble understanding why need to reorder tags and at resolver layer, how would the clients provide reorder input? coz the schema for suggested tags only has this line: https://github.com/Pocket/list-api/blob/main/schema.graphql#L119

Reply:
having trouble understanding why need to reorder tags and at resolver layer, how would the clients provide reorder input? coz the schema for suggested tags only has this line: https://github.com/Pocket/list-api/blob/main/schema.graphql#L119

edit: synced with kelvin, we need to preserve the order for the data loader